src/components/tabs.js

```diff
@@ -0,0 +1,119 @@
+(() => ({
+  name: 'Tabs',
+  type: 'CONTENT_COMPONENT',
+  allowedTypes: ['TAB_COMPONENT'],
+  orientation: 'HORIZONTAL',
+  jsx: (() => {
+    const { Tabs, Tab } = window.MaterialUI.Core;
+    const { Icons } = window.MaterialUI;
+    const { Children, env } = B;
+    const {
+      defaultValue,
```

Comment:
Can we add a check that the defaultValue actually exists? 

Now it is possible to fill in the value 1000 and it will display nothing because there are only 2 tabs for example.

Make a fallback that it shows the latest tab that exists